packaging/nnstreamer.spec

```diff
@@ -736,6 +744,12 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 
 %postun -p /sbin/ldconfig
 
+%post test-devel
+mv /etc/nnstreamer.ini /etc/nnstreamer.ini.bak
+
+%postun test-devel
+mv /etc/nnstreamer.ini.bak /etc/nnstreamer.ini
```

Comment:
Be careful with post/postun. If it's upgrade, not uninstall, this may go wrong.

Reference: https://stackoverflow.com/questions/7398834/rpm-upgrade-uninstalls-the-rpm

Reply:
Removed this part, and a new commit to resolve this issue, 
[PTAL](https://github.com/nnstreamer/nnstreamer/pull/3128/commits/9b4214881c13d0bad614df28bb6ef745bb71beba) :)